library/L2_HAL/actuators/servo/test/servo_test.cpp

```diff
@@ -0,0 +1,78 @@
+#include "L0_Platform/lpc40xx/LPC40xx.h"
+#include "L2_HAL/actuators/servo/servo.hpp"
+#include "L4_Testing/testing_frameworks.hpp"
+
+namespace sjsu
+{
+EMIT_ALL_METHODS(Servo);
+
+TEST_CASE("Testing Servo", "[Servo]")
+{
+  // Make a mock pin to work with
+  Mock<Pwm> mock_pwm;
+
+  // Fake the implementation of:
+  // Initialize
+  // SetFrequency
+  // SetDutyCycle
+  Fake(Method(mock_pwm, Initialize));
+  Fake(Method(mock_pwm, SetFrequency));
+  Fake(Method(mock_pwm, SetDutyCycle));
+
+  // Retrieve a reference to the Pin to be injected as the return value
+  // of Gpios's GetPin() method.
+  Pwm & test_pwm = mock_pwm.get();
+
+  // Inject test_gpio into button object
+  Servo test_servo(test_pwm);

```

Comment:
Suggestion: You could also pass this directly in via:

```suggestion
  Servo test_servo(mock_pwm.get());
```